src/core/lib/transport/timeout_encoding.cc

```diff
@@ -95,8 +95,9 @@ int grpc_http2_decode_timeout(grpc_slice text, grpc_millis* timeout) {
   const uint8_t* end = GRPC_SLICE_END_PTR(text);
   int have_digit = 0;
   /* skip whitespace */
-  for (; p != end && *p == ' '; p++)
+  for (; p != end && *p == ' '; p++) {
     ;
```

Comment:
No need for this anymore, now that there are braces around it.

Reply:
Done

Reply:
Doesn't look like this was actually done.